presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -124,4 +127,18 @@ public BaseJdbcConfig setCaseInsensitiveNameMatchingCacheTtl(Duration caseInsens
         this.caseInsensitiveNameMatchingCacheTtl = caseInsensitiveNameMatchingCacheTtl;
         return this;
     }
+
+    @NotNull
+    public UnsupportedTypeHandlingStrategy getUnsupportedTypeHandlingStrategy()
+    {
+        return unsupportedTypeHandlingStrategy;
+    }
+
+    @Config("unsupported-type.handling-strategy")
```

Comment:
`unsupported-type.handling-strategy=FAIL` is not nearly as clear as `fail-on-unsupported-types=true`.

I understand the enum is a preparation for an option to map unsupported columns to varchar (see also https://github.com/prestosql/presto/pull/186)
I think the option name needs to be revisited.